### PR TITLE
[GT-133] Adds contact section to the guide page

### LINKED ISSUE #149: Add contact address to help page

### PR DESCRIPTION
**Pull request by rowan04**
- in the guide page, adds a contact section containing the apel admins email address

I have no idea why so many lines were edited

Resolves #149 hopefully

contactdashboardchange

### CODE REVIEW

File: dashboards/iris/guide.json [hunk 1]
```diff
@@ -150,5 +290,6 @@
   "timezone": "",
   "title": "Guide",
   "uid": "3FEa2VaZz",
-  "version": 13
-}
+  "version": 1,
+  "weekStart": ""
+}
```
[the remark this answers is not in the document]
Comment (rowan04) [reply dashboards/iris/guide.json:295]: ok

File: dashboards/iris/guide.json [hunk 2]
```diff
@@ -3,126 +3,266 @@
...
     },
     {
-      "content": "\n## Filters\n\nFilters (or variables as they are called by Grafana) on this dashboard can be used to choose a subset of Sites and VOs to view in the graphs and charts.\n\n![filters](/public/img/filters.png)\n\nYou can also choose to select all the Grid or all the Cloud Sites or VOs using the 'Tags' feature:\n\n![tags](/public/img/tags.png)\n",
+      "datasource": {
+        "type": "datasource",
+        "uid": "grafana"
+      },
       "gridPos": {
-        "h": 15,
+        "h": 16,
```
[the remark this answers is not in the document]
Comment (rowan04) [reply dashboards/iris/guide.json:106]: i think this was me, as the panel had a scroll bar, but if it was one pixel longer it didn't need the scroll bar. I think i changed it and didn't think much of it, maybe thinking i had shortened it by accident in the first place

File: dashboards/iris/guide.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (rowan04) [reply dashboards/iris/guide.json]: is it even there now? I definitely added it in that commit - maybe vscode is removing it when i push?

[the remark this answers is not in the document]
Comment (tofu-rocketry) [reply dashboards/iris/guide.json]: Yup, that's worked.

**Review by tofu-rocketry — APPROVED** [review]

**Pull request merged**